proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for correct receiving of system time.
+
+
+## Motivation
+
+Allow SDL Core obtain time from the system in UTC format. Exclude possibility of UTC time distortion in case using system time from other components of the system.
+
+
+## Proposed solution
+
+
+The solution proposes adding new RPC between SDL and HMI.  GetSystemTime gives possibility to get the System Time status by making the following additions to the HMI API.

```

Comment:
@AStasiuk 
- just 'system time' or 'real system time', no need to say that aloud )
- better ".. by introducing new API.." or ".. extending HMI API with.."

Reply:
@dev-gh Fixed in  https://github.com/AStasiuk/sdl_evolution/pull/1/commits/280d2cb602d95cd086a87fca94746cddd3831b6f